installer/templates/new/web/web.ex

```diff
@@ -48,9 +48,11 @@ defmodule <%= application_module %>.Web do
     end
   end
 
-  def view do
+  def view(opts \\ []) do
```

Comment:
You can pass the options in your own app by modifying web.ex to suit your needs or calling `use Phoenix.View` directly with the options you need